apps/web/src/pages/SearchRequests/ViewSearchRequestPage/components/UpdateSearchRequest.tsx

```diff
@@ -28,6 +27,18 @@ import useRoutes from "~/hooks/useRoutes";
 
 type FormValues = UpdatePoolCandidateSearchRequestInput;
 
+const UpdateSearchRequestOptions_Query = graphql(/* GraphQL */ `
+  query UpdateSearchRequestOptions {
+    statuses: localizedEnumStrings(enumName: "PoolCandidateSearchStatus") {
+      value
+      label {
+        en
+        fr
+      }
+    }
+  }
+`);
+
```

Comment:
I found a bug with this form. When the user refreshes the request page the `statuses` are fetched in this component, therefore it doesn't show the correct value in the select box. Since the options query hasn't completed yet. You'll need to turn this into a fragment and move it into `pages/SearchRequests/ViewSearchRequestPage.tsx`

Reply:
Should be fixed in [fix statuses default value issue](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10701/commits/f023ff33f5dafa2faeea7ce866bdc2538c746e9e)

Reply:
Looks good :+1: